crates/breez-sdk/core/src/sdk.rs

```diff
@@ -1179,6 +1179,31 @@ impl BreezSdk {
             payment: payment_result?,
         })
     }
+
+    /// Returns the metadata for the given token identifiers.
+    ///
+    /// Results are not guaranteed to be in the same order as the input token identifiers.    
+    ///
+    /// If the metadata is not found locally in cache, it will be queried from
+    /// the Spark network and then cached.
+    pub async fn get_tokens_metadata(
```

Comment:
I feel like it should be `list_tokens_metadata`, so it fits with the listing payments, unclaimed desposits and fiat currencies. What do you think?